service/history/mutableStateBuilder.go

```diff
@@ -186,6 +186,7 @@ func newMutableStateBuilder(
 	logger log.Logger,
 	namespaceEntry *cache.NamespaceCacheEntry,
 	startTime time.Time,
+	firstWorkflowTaskBackoff *time.Duration,
```

Comment:
`firstWorkflowTaskBackoff *time.Duration` should not be part of the initialization logic

Reply:
the reason start time is part of initialization parameter if for workflow reset case, where start time is the reset time, not the timestamp of start event